mavlink/modules/flight_controller.py

```diff
@@ -141,7 +141,7 @@ def upload_commands(self, commands: "list[dronekit.Command]") -> bool:
 
     def upload_land_command(self, latitude: float, longitude: float) -> bool:
         """
-        Given a target latitude and longitude, overwrite the drone's current mission 
+        Given a target latitude and longitude, overwrite the drone's current mission.
         with a corresponding dronekit land command.
```

Comment:
Is this supposed to be in the same sentence?

Reply:
Oh good catch!